addon/store.js

```diff
@@ -46,45 +46,39 @@ var Store = Ember.Object.extend({
       this.set("array", {});
     },
     clear(type) {
-        if(type === undefined) {
+        if (type === undefined) {
             this.reset();
         }
+
         delete this.get("identityMap")[type];
         arrayForType(type, this).clear();
-
-        run.once(() => {
```

Comment:
This is scheduling an anonymous function to run once, which won't work as far as I know.

For example, if you invoke clear 3 times, scheduleUpdate will be invoked 3 times.  So I don't believe wrapping this in an Ember.run.once will do anything.  But I'm not a runloop expert, it's just what I think is happening and then validated it with some snippets.